packages/fluentui/react-northstar/src/themes/teams/components/Button/buttonStyles.ts

```diff
@@ -23,7 +23,7 @@ export const buttonStyles: ComponentSlotStylesPrepared<ButtonStylesProps, Button
 
     return {
       height: v.height,
-      minWidth: _.isNil(p.loading) ? v.minWidth : v.loadingMinWidth,
+      minWidth: _.isNil(p.loading) || !p.loading ? v.minWidth : v.loadingMinWidth,
```

Comment:
```suggestion
      minWidth: p.loading ? v.loadingMinWidth : v.minWidth,
```

I am not sure about usage of `_.isNil()`. Can you please check?

Reply:
It returns true when undefined or null. So your change actually makes more sense.